slack_sdk/web/deprecation.py

```diff
@@ -10,8 +10,10 @@
     "admin.conversations.whitelist.",
 ]
 
+deprecated_method_prefixes_2023_07 = ["stars."]
```

Comment:
Wondering if we want to include a comment to the changelog above this deprecation too? It's already included in the comment below so not a huge deal if not!

Reply:
Sadly, we don't have any workaround for this breaking change. So, I cannot think of any good comment for it